schema/tx_schema.md

```diff
@@ -0,0 +1,155 @@
+## Overview
+
+Signal size can be automatically calculated or explicitly specified. Cannot (currently) specify signal position.

```

Comment:
Are there any situation where you would like to be explicit about the signal position?

Reply:
it seems like the inverter signals are fully packed so it may not be necessary BUT probably good to support this actually

the issue is that specifying only a subset of a message's signals in positions and leaving the rest of the signals unallocated creates a need for a relatively complex algorithm that optimally packs around the fixed signals... recipe for extreme pain

i think the nice solution is to require all positions specified or no positions specified

- [support specifying signal position]

Reply:
> it seems like the inverter signals are fully packed

this is not true - this inverter message is very not packed (see bits 33-40, or lack thereof)
```
BO_ 32 DCM_INVL_Command_Message: 8 DCM
SG_ Inverter_Enable_INVL : 40|1@1+ (1,0) [0|1] "Bit" INVL
SG_ Direction_Command_INVL : 32|1@1+ (1,0) [0|1] "Bit" INVL
SG_ Speed_Command_INVL : 16|16@1- (1,0) [-32768|32767] "rpm" INVL
SG_ Torque_Command_INVL : 0|16@1- (0.1,0) [-3276.8|3276.7] "Nm" INVL,FSM
SG_ Inverter_Discharge_INVL : 41|1@1+ (1,0) [0|1] "Bit" INVL
SG_ Torque_Limit_Command_INVL : 48|16@1- (0.1,0) [-3276.8|3276.7] "Nm" INVL
SG_ Speed_Mode_Enable_INVL : 42|1@1+ (1,0) [0|1] "Bit" INVL
SG_ RollingCounter_INVL : 44|4@1+ (1,0) [0|15] "Bits" INVL
```

talked to @pasitlaothamatas and he agrees with:
> require all positions specified or no positions specified

Reply:
+1 on all packed or no packed